docs/node-configuration.md

```diff
@@ -76,6 +77,8 @@ where:
    messages to just 10 of them. With BroadcastFactor set to 100 it will always send messages
    to all peers, any value in-between 0 and 100 is used for weighted calculation, for example
    if it's 30 then 13 neighbors will be used in the previous case.
+- `BroadcastTxsBatchTime` (`Duration`) is the time for collecting transactions in a batch for
```

Comment:
> the time for collecting transactions in a batch

the time limit for collecting batch of transactions ?

Reply:
as you wish